test/integration/main_test.go

```diff
@@ -358,6 +358,149 @@ deletion_timeline:
 	}
 }
 
+// TestMappingReusableWorkflowCall tests the mapping file-copy reusable workflow has successfully
+// uploaded the file to gcs and triggered pmap event handler, and write the corresponding entry
+// into bigquery.
+func TestMappingReusableWorkflowCall(t *testing.T) {
+	t.Parallel()
+	cases := []struct {
+		name                string
+		bigqueryTable       string
+		traceID             string
+		wantResourceMapping *v1alpha1.ResourceMapping
+	}{
+		{
+			name:          "mapping-copy-success-event",
+			bigqueryTable: cfg.MappingTableID,
+			traceID:       "fake-pmap-dev-manual-test-mapping",
+			wantResourceMapping: &v1alpha1.ResourceMapping{
+				Resource: &v1alpha1.Resource{
+					Provider: "gcp",
+					Name:     fmt.Sprintf("//storage.googleapis.com/%s", "pmap-static-ci-bucket-7faa"),
+				},
+				Contacts: &v1alpha1.Contacts{Email: []string{"pmap.mapping@gmail.com"}},
+			},
+		},
+	}
+
+	for _, tc := range cases {
+		tc := tc
+
+		t.Run(tc.name, func(t *testing.T) {
+			t.Parallel()
+
+			ctx := context.Background()
+
+			t.Logf("using trace ID %s", tc.traceID)
+			t.Logf("using workflow run ID %s", cfg.WorkflowRunID)
+			// Check if the file uploaded exists in BigQuery.
+			queryString := fmt.Sprintf("SELECT * FROM `%s.%s.%s`", cfg.ProjectID, cfg.BigQueryDataSetID, tc.bigqueryTable)
+			queryString += `WHERE JSON_VALUE(data.githubSource.workflowRunId) = ?`
+
+			bqQuery := bqClient.Query(queryString)
+			bqQuery.Parameters = []bigquery.QueryParameter{{Value: cfg.WorkflowRunID}}
```

Comment:
Do you need this when you use queryString as WHERE JSON_VALUE(data.githubSource.workflowRunId) = ?